### PR TITLE
Adjust cucushift e2e default skip tags

### PR DESCRIPTION
**Pull request by liangxia**
Remove those tags from default skip tags as we may need to execute some of the tests that contain the tag. Also we may want to add new jobs specific on those tests.

/cc @jhou1 @JianLi-RH @dis016 @pruan-rht 

We will need to update/fix existing jobs in follow up PRs due to this change.

### CODE REVIEW

**Comment by liangxia** [comment]
Those failures are expected. We will fix them in individual job level

File: ci-operator/step-registry/cucushift/e2e/cucushift-e2e-ref.yaml [hunk 1]
```diff
@@ -20,14 +20,10 @@
     default: "@aws-ipi"
     documentation: "Executed cucumber tags"
   - name: E2E_SKIP_TAGS
-    default: "not @console
```
Comment (jhou1) [inline ci-operator/step-registry/cucushift/e2e/cucushift-e2e-ref.yaml:23]: Can we trust console cases now?

Comment (liangxia) [reply ci-operator/step-registry/cucushift/e2e/cucushift-e2e-ref.yaml:23]: @jhou1  According to the test results in https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/27625/rehearse-27625-periodic-ci-openshift-verification-tests-master-stable-4.9-e2e-aws-ipi/1511867092322750464, I can see many console test cases passed.

Comment (jhou1) [reply ci-operator/step-registry/cucushift/e2e/cucushift-e2e-ref.yaml:23]: Thanks!

**Pull request merged**